### PR TITLE
Create package.json

### PR DESCRIPTION
**Pull request by yatharthshahjpr**
PLEASE MERGE THE LAST TWO PR'S THEY ARE BOTH REQUIRED FOR THE CODE TO RUN USING JS

### CODE REVIEW

**Comment by yatharthshahjpr** [comment]
@AdiChat , Please review my PR! :)

File: code/data_structures/kd_tree/kd_tree/kd_tree_in_js/package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "kd-tree-javascript",
+  "version": "1.0.3",
+  "description": "",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ubilabs/kd-tree-javascript.git"
```
Comment (AdiChat) [inline code/data_structures/kd_tree/kd_tree/kd_tree_in_js/package.json]: Looks like you copied the code. This cannot be merged 👍

**Comment by AdiChat** [comment]
We have informed you previously to keep all concerned files in a single pull request. There is no point in making multiple pull requests for the same thing. Kindly rectify 👍

File: code/data_structures/kd_tree/kd_tree/kd_tree_in_js/package.json (hunk 1 above)
Comment (yatharthshahjpr) [reply code/data_structures/kd_tree/kd_tree/kd_tree_in_js/package.json]: For the JSON object creation it was done! :)
Still I will make furtheer changes for the same! 💯

**Comment by yatharthshahjpr** [comment]
Ok ............I actually make this error again and again in  hurry ....really sorry for it! 👍

**Pull request closed without merging**